app/src/test/java/com/husseinelfeky/moviesexplorer/master/MasterViewModelTest.kt

```diff
@@ -80,7 +82,8 @@ class MainViewModelTest : AutoCloseKoinTest() {
 
         // THEN - Verify that search results are found and contain 8 differentiable items
         // (2 years and 6 movies).
-        assertThat(result.isNullOrEmpty(), `is`(false))
+        assertThat(result, `is`(notNullValue()))
+
         assertThat(result.size, `is`(8))
```

Comment:
It would actually be better to remove this assertion completely, since the result can never be null and we already assert the list size right below.

```suggestion
        assertThat(result.size, `is`(8))
```

Reply:
Yes you are right